app/views/answer/_quiz_page.html.erb

```diff
@@ -9,17 +10,15 @@
           <b><li><%= question.body %></li></b>
           <% question.options.each do |option| %>
             <ul>
-            
-            <%= form.radio_button("answers[#{question.id}]", option.option1) %>
-            <%= label_tag "#{option.option1}", option.option1 %>
+            <%= form.radio_button("answers[#{question.id}]", option.option) %>
+            <%= label_tag "#{option.option}", option.option %>
```

Comment:
option.option seems redundant like we discussed yesterday, can we change this field name?